app/assets/stylesheets/modules/review.scss

```diff
@@ -0,0 +1,51 @@
+@import "variables";
+@import "mixins";
+
+#reviews-column {
+  @include padding;
+
+  .reviews {
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.